quantum/split_common/split_util.h

```diff
@@ -7,6 +7,9 @@
 #include <stdlib.h>
 #include "eeconfig.h"
 
+// backlight level store index in serial_master_buffer[] for slave to read
+#define SERIAL_BACKLIT_START   0x00
```

Comment:
Shouldn't this be left in serial.h?

Reply:
see 
https://github.com/qmk/qmk_firmware/pull/4419#issuecomment-438721337